.github/workflows/connectors_qa_checks.yml

```
@@ -0,0 +1,60 @@
+name: Connector QA Checks
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - "airbyte-integrations/connectors/**/*"
+
+jobs:
+  generate-matrix:
+    name: Generate Connectors Matrix
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout Current Branch
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Generate Connectors Matrix from Changes
+        id: generate-matrix
+        run: |
+          # Get the list of modified connectors
+          echo "connectors_matrix=$(./poe-tasks/get-modified-connectors.sh --json)" | tee -a $GITHUB_OUTPUT
+    outputs:
+      connectors-matrix: ${{ steps.generate-matrix.outputs.connectors_matrix }}
+
+  qa-checks:
+    if: github.event.pull_request.draft == false
+    needs: [generate-matrix]
+    strategy:
+      matrix: ${{ fromJson(needs.generate-matrix.outputs.connectors-matrix) }}
+      max-parallel: 5 # Limit number of parallel jobs
+      fail-fast: false # Don't stop on first failure
+    name: QA Checks for ${{ matrix.connector || ' (no connectors modified)' }}
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install uv
+        uses: astral-sh/setup-uv@v6
+      - name: Install QA Checks
+        run: |
+          uv tool install airbyte-ci/connectors/connectors_qa
+      - name: Run QA Checks
+        # remove "-strict-encrypt" suffix, if present, for parity with the previous version of this workflow
+        run: |
+          connector_name=${{ matrix.connector }}
+          connectors-qa run --name ${connector_name%-strict-encrypt}
```

When no connectors are changed, the latest version of `get-modified-connectors.sh` should return one empty-string entry. So, we should make these steps a no-op if they receive `matrix.connector == ""`.

```suggestion
      - name: Checkout Airbyte
        if: matrix.connector
        uses: actions/checkout@v4
        with:
          fetch-depth: 0
      - name: Install uv
        if: matrix.connector
        uses: astral-sh/setup-uv@v6
      - name: Install QA Checks
        if: matrix.connector
        run: |
          uv tool install airbyte-ci/connectors/connectors_qa
      - name: Run QA Checks
        if: matrix.connector
        # remove "-strict-encrypt" suffix, if present, for parity with the previous version of this workflow
        run: |
          connector_name=${{ matrix.connector }}
          connectors-qa run --name ${connector_name%-strict-encrypt}
```